src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -26,58 +26,86 @@ import { AcceptPaymentMethod } from 'claimant-response/form/models/acceptPayment
 function createDraftClaimantResponseForFullRejection (): DraftClaimantResponse {
   const draftResponse: DraftClaimantResponse = new DraftClaimantResponse()
   draftResponse.settleAdmitted = new SettleAdmitted(YesNoOption.NO)
-  draftResponse.paidAmount = new PaidAmount(PaidAmountOption.NO,10,100)
+  draftResponse.paidAmount = new PaidAmount(PaidAmountOption.NO, 10, 100)
   return draftResponse
 }
+
 function createDraftClaimantResponseBaseForAcceptance (accept: YesNoOption): DraftClaimantResponse {
   const draftResponse: DraftClaimantResponse = new DraftClaimantResponse()
   draftResponse.settleAdmitted = new SettleAdmitted(YesNoOption.YES)
-  draftResponse.paidAmount = new PaidAmount(PaidAmountOption.YES,10,100)
+  draftResponse.paidAmount = new PaidAmount(PaidAmountOption.YES, 10, 100)
   draftResponse.acceptPaymentMethod = new AcceptPaymentMethod(accept)
   return draftResponse
 }
 
-const courtPaymentIntention = {
+const paymentIntentionInInstallment = {
   paymentOption: PaymentOption.INSTALMENTS,
   paymentDate: MomentFactory.currentDate().add(5, 'days'),
-  repaymentPlan:  {
+  repaymentPlan: {
     instalmentAmount: 10,
     firstPaymentDate: MomentFactory.currentDate().add(30, 'days'),
     paymentSchedule: PaymentSchedule.EVERY_MONTH,
     completionDate: MomentFactory.currentDate().add(300, 'days'),
     paymentLength: '3 Months'
-  }}
+  }
+}
+
+const installmentPaymentIntention = {
+  'paymentOption': 'INSTALMENTS',
+  'paymentDate': MomentFactory.currentDate().add(5, 'days'),
+  'repaymentPlan': {
+    'instalmentAmount': 10,
+    'firstPaymentDate': MomentFactory.currentDate().add(30, 'days'),
+    'paymentSchedule': 'EVERY_MONTH',
+    'completionDate': MomentFactory.currentDate().add(300, 'days'),
+    'paymentLength': '3 Months'
```

Comment:
I know this is just test data, but would be nice to have reasonable length based on completionDate. 300 days in future are not 3 months. :)